### DISCUSSION TITLE
SQ3 Surface Pro 9 New Teams and Outlook not working.

### DISCUSSION THREAD

**Discussion by AngieMKim** [MicrosoftEdge/WebView2Feedback#5020] [Q&A]
Error "We've run into an issue. We can't find a required component to run Teams. Download and install it and then try opening Teams again.

Upon trying to Download Edge WebView2 the error Installation failed. The Microsoft Edge WebView2 Runtime is already installed for the system.

Repair of from control panel still does not solve the issue.
![New MSTeams We have run into an issue](https://github.com/user-attachments/assets/70ca7531-630b-45db-9818-bf25ae12e810)
![WebView2 Installation Failed](https://github.com/user-attachments/assets/7f2fb8a3-a194-483e-a0b1-9c1c29305854)

**Comment by sivMSFT** [comment]
Hi @AngieMKim,

Thank you for reporting the issue and providing detailed information.

We apologize for the inconvenience this has caused. Our team is currently investigating the problem, and we are working diligently to find a solution.

In the meantime, could you please let us know if the issue has been resolved on your end? Your feedback is important to us and will help in our investigation.
